### ISSUE TITLE
Extra colours

### ISSUE THREAD

**Issue by smalltimer** [ful1e5/Bibata_Cursor#136]
Hi, 

I observed that the Bibata Extra library was archived last year. Does that mean the extra colours will be moved to this repo, or that the extra colours are being dropped and will not be included in future versions of Bibata ? 

Thank you!

**Comment by ful1e5** [comment]
I have made the decision to archive the [Bibata Extra](https://github.com/ful1e5/Bibata_Extra_Cursor) project, as the current state of Bibata now offers customization options and support for Windows cursors. When I initially created the Bibata Extra repository, my goal was to address the most frequently requested color options from users who were not involved in cursor development. At the time, I was a student and cursor development was a passion and part-time job for me. However, as other open-source projects have taken up more of my time and scheduling tasks have become increasingly difficult, I have decided to archive certain projects in order to conserve my efforts and use them to create other tools and make meaningful contributions to the community.

In addition, I am planning to develop a new project based on Bibata that will enable personalization and downloading through a web-app. I will provide further information about this in the README.md file when it becomes available.

I hope this information is useful for you.
